raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -1596,7 +1596,7 @@ contract TokenNetwork is Utils, Controllable {
         uint256 channel_identifier,
         address participant,
         uint256 total_withdraw,
-        uint256 expiration_timestamp,
+        uint256 recoverable_until,
```

Comment:
This function is called directly with parameters from `WithdrawInput`, therefore I think the name here should be the same from the struct:
```suggestion
        uint256 withdrawable_until,
```

Reply:
Nice catch ✅